MIGRATION.md

```diff
@@ -2,6 +2,27 @@
 
 ### Migrating from v2 to v3
 
+Confirm the `o-teaser-collection__heading--inverse` modifier is used with `o-teaser-collection__heading`. I.e. `o-teaser-collection__heading o-teaser-collection__heading--inverse`. This was optional before.
+
+`oTeaserCollection` has changed, it now outputs all `o-teaser-collection` css by default. See the README for `oTeaserCollection` options to include css granularly.
```

Comment:
Would making README a link work here?
```suggestion
`oTeaserCollection` has changed, it now outputs all `o-teaser-collection` css by default. See the [README](./README.md) for `oTeaserCollection` options to include css granularly.
```

Reply:
Not current, not in the registry. We could add support though. https://github.com/Financial-Times/origami-registry-ui/issues/170

Reply:
Oh wait but also yeah, that's fine, because we don't show the migration guide outside of github at the moment.